client/my-sites/exporter/index.jsx

```diff
@@ -10,6 +10,7 @@ import { connect } from 'react-redux';
 import config from 'config';
 import QuerySiteGuidedTransfer from 'components/data/query-site-guided-transfer';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteType } from 'state/ui/selectors';
```

Comment:
This can be grouped together:
```javascript
import { getSelectedSiteId, getSelectedSiteType } from 'state/ui/selectors';
```

See also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import which is similar to, but annoyingly different from destructure syntax.